pkg/controller/node/nodecontroller.go

```diff
@@ -191,6 +197,8 @@ func NewNodeController(
 		allocateNodeCIDRs:         allocateNodeCIDRs,
 		forcefullyDeletePod:       func(p *api.Pod) error { return forcefullyDeletePod(kubeClient, p) },
 		nodeExistsInCloudProvider: func(nodeName string) (bool, error) { return nodeExistsInCloudProvider(cloud, nodeName) },
+		computeZoneStateFunc:      ComputeZoneState,
+		zoneStates:                map[string]zoneState{},
```

Comment:
nit: we should be consistent and create all map either by:
map[xxx]yyy{}
or by:
make(map[xxx]yyy)
?